### DISCUSSION TITLE
bug with datagrid

### DISCUSSION THREAD

**Discussion by Everyyx** [AvaloniaUI/Avalonia#12119] [General]
i've installed the datagrid 11.0 follow these steps:
dotnet add package Avalonia.Controls.DataGrid
include styles created specifically for that:
app.axaml
<Application.Styles>
<FluentTheme Mode="Light" />
<StyleInclude Source="avares://Avalonia.Controls.DataGrid/Themes/Fluent.xaml"/>
</Application.Styles>
but when add datagrid in my panel:
<datagrid ite....>there only itemsource to choses,if i add property items,the avalonia debug output said:"Unable to resolve suitable regular or attached property Items on type Avalonia.Controls.DataGrid:Avalonia.Controls.DataGrid ",so, where did I go wrong

**Comment by timunie** [comment]
in 11.0 Items is not ItemsSource. Please read this guide carefully: https://docs.avaloniaui.net/docs/next/stay-up-to-date/upgrade-from-0.10
